src/spu/interface.h

```diff
@@ -134,6 +137,22 @@ class impl final : public SPUInterface {
     uint8_t *pSpuBuffer;
     uint8_t *pMixIrq = 0;
 
+
+    struct CaptureBuffer {
+        static const int CB_SIZE = 1024 * 8;
+        // These buffers have to be large enough to allow the CD-XA to stream in enough data.
+        uint16_t CDCapLeft[CB_SIZE] = {0};
```

Comment:
So I think those would require to be part of the savestate to be properly migrated over during, well, a savestate. Check out spu/freeze.cc and core/sstate.h around line 116 for more on this. It might be a little bit intimidating to modify, so let me know if you need help for this.